public/locale/de.json

```diff
@@ -99,6 +99,9 @@
   "none": "keine",
   "remove": "entfernen",
   "close": "schließen",
-  "deprecation": "Warnung: Dieser Knoten ist veraltet, und wird demnächst nicht mehr unterstützt. Mehr Infos unter <a href='https://openwrt.org/supported_devices/432_warning'>4/32 warning</a>. <br> Wenn du der Eigentümer des Gerätes bist, bitten wir dich, das Gerät zu ersetzen, um weiterhin am Netz teilnehmen zu können.",
+  "deprecation": "deprecated",
+  "deprecation-text": "Warnung: Dieser Knoten ist veraltet, und wird in Zukunft nicht mehr unterstützt. Mehr Infos unter <a href='https://openwrt.org/supported_devices/864_warning'>4/32 warning</a>. <br> Wenn du der Eigentümer des Gerätes bist, bitten wir dich, das Gerät zu ersetzen, um weiterhin am Netz teilnehmen zu können.",
```

Comment:
also just as a comment:
how will we handle deprecation notices for devices that aren't related to flash or ram being limited?
I mean this text is only kind of accurate, the hard limit for the devices currently listed is mostly the amount of flash that is available.
Example: devices from the Lantiq target (xrx200 subtarget) could've ended up deprecated, the same goes for the WDR4900 in 2022. These were brought back, but it's not always flash and ram at play. Sometimes it's bugs in the target caused by changes in the Kernel that downstream needs to adjust to. Or that the bootloader has issues booting a larger Kernel than 4MiB even though the flash itself is larger.

^this may be addressed later, but I wanted to bring it up :)

Reply:
This is an example text, that should be adjusted by anyone who uses it anyway.

Reply:
ok 👍
but maybe we should atleast fix it, so it says 8/64 correctly. instead of 4/32

Reply:
Done!